pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.40</version>
+    <version>3.56</version>
```

Comment:
Needed to update some dependencies and match with JCasC

Reply:
and this is good, to get the latest test-harness.

Reply:
indeed that's one of the dependencies to be updated. `configuration-as-code:1.35` forces the latest version of test-harness. But updating also forced me to exclude some transitive dependencies of powermock which differ from the one used in configuration-as-code.